src/pages/OtakuFR/tests.json

```diff
@@ -1,27 +1,27 @@
-{
-  "title": "OtakuFR",
-  "url": "https://www.otakufr.com/",
-  "testCases": [
-    {
-      "url": "https://www.otakufr.com/Ahiru-no-Sora-Vostfr/21/",
-      "expected": {
-        "sync": true,
-        "title": "Ahiru no Sora",
-        "identifier": "Ahiru-no-Sora-Vostfr",
-        "overviewUrl": "https://www.otakufr.com/Ahiru-no-Sora-Vostfr/",
-        "nextEpUrl": "https://www.otakufr.com/Ahiru-no-Sora-Vostfr/22/",
-        "episode": 21,
-        "uiSelector": false
-      }
-    },
-    {
-      "url": "https://www.otakufr.com/Ahiru-no-Sora-Vostfr/",
-      "expected": {
-        "sync": false,
-        "title": "Ahiru no Sora",
-        "identifier": "Ahiru-no-Sora-Vostfr",
-        "uiSelector": true
-      }
-    }
-  ]
-}
+{
+  "title": "OtakuFR",
+  "url": "https://otakufr.co",
+  "testCases": [
+    {
+      "url": "https://otakufr.co/episode/ahiru-no-sora-21-vostfr/",
+      "expected": {
+        "sync": true,
+        "title": "Ahiru no Sora",
+        "identifier": "ahiru-no-sora",
+        "overviewUrl": "https://otakufr.co/anime/ahiru-no-sora/",
+        "nextEpUrl": "https://otakufr.co/episode/ahiru-no-sora-21-vostfr/",

```

Comment:
Need to set next episode to 22 instead of 21 in "nextEpUrl" line.

`        "nextEpUrl": "https://otakufr.co/episode/ahiru-no-sora-22-vostfr/",`